pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -13,7 +13,7 @@
 {{- end }}
 
 <!-- Create resource -->
-## Create a {{ .Header.Title }} Resource {#create}
+## Create {{ .Header.Title }} Resource {#create}
```

Comment:
This works by itself, but it’s be misaligned with the lookup heading below it. If we did this, I’d also suggest we tweak “Look up an existing…” to read similarly: e.g., “Look up existing...” Alternatively, we could change this heading to do what that one does, and just add a word, so it always works: “Create a new…”, say. This way, we’d end up with either “Create Bucket Resource” and “Look up Existing Bucket Resource” or “Create a new Bucket Resource” and “Look up an existing Bucket resource”, both of which work. (Though I'd personally prefer the latter.)

```suggestion
## Create a New {{ .Header.Title }} Resource {#create}
```

Reply:
an will always be correct on the lookup because it is proceeding a fixed word: existing, but i removed it to satisfy the alignment requirement.